### PR TITLE
[French] Added french locals

### PR DESCRIPTION
**Pull request by PierreGrillet**
Added french locals to Belgium, Luxembourg, Switzerland, Monaco and Canada (Québec).

### CODE REVIEW

File: Langs/080C.lang [hunk 1]
```diff
@@ -0,0 +1,79 @@
+[MetaData]
+Language=French - Belgium
+Translator=PierreGrillet; Nits77; villisek; BSoD38
```
Comment (micwoj92) [inline Langs/080C.lang:3]: Is it needed to add all translators to this file? What if they don't know that version of French and there are some subtle differences?

Comment (PierreGrillet) [reply Langs/080C.lang:3]: Frankly I just copy paste the French France translation (and their authors), I don't think they will be such difference, as translation can already been discussed. (Should we translate that or not ? Is that not to far fetch ? etc...)

The french spoken in the bordering countries is very close anyway, so in a technical environment like computing, I don't think they is much difference.

However, for Quebec French some major difference can exist, as they are more strict and translate more terms (in general) (Never tell a Quebecer that he live in Canada :wink:, that the best way to not understand them).

My best bet, is to wait for an review by a Quebec french speaker. Maybe for other.

Asking review from any french speaker, or from @Nits77, @villisek, @vincent64. :+1:  :smile:

**Comment by micwoj92** [comment]
@krystian3w why did you downvote?

**Comment by krystian3w** [comment]
Because  better create shorter name "FR origin team" instead of nick names.

**Comment by micwoj92** [comment]
I mean, it looks fine to me. If for some reason a version of French has different version of word then feel free to only update that version

**Pull request merged**